### PR TITLE
Kokos

### PR DESCRIPTION
**Pull request by ManosKokos**
Added config.json in src, solved merge conflicts

### CODE REVIEW

File: src/Controller/Controller.js [hunk 1]
```diff
@@ -131,75 +153,123 @@ function insert_pad_id_toDB(db, id, name, ip) {
...
 		console.log("Pad not found");
 		res.status(404);
 		return;
 	}
+	
 	PadMap.get(req.body.id).name = req.body.name;
 	var json_ret = PadMap.get(req.body.id);
+	
+	var db = Database_Connect();	
+	// REMEMBER TO CLOSE CONNECTION TO DATABASE!!!!
```
Comment (ManosKokos) [inline src/Controller/Controller.js]: @parriiss Remeber to close connection to mysql!!!

File: src/Controller/Controller.js [hunk 2]
```diff
@@ -131,75 +153,123 @@ function insert_pad_id_toDB(db, id, name, ip) {
 	});
 	db.end();
```
Comment (ManosKokos) [inline src/Controller/Controller.js]: @parriiss Not a good place to close db. Please close the connection at the same scope it opens

File: src/Controller/Controller.js [hunk 3]
```diff
@@ -131,75 +153,123 @@ function insert_pad_id_toDB(db, id, name, ip) {
...
+/**
+ * Changes the name of a pad in the database 
+ * 
+ * @param {object} db 		Holds an already open connection to the database  
+ * @param {string} padId 	String of the id of the pad to be editted
+ * @param {string} newName 	String of the new name for pad  
+ * 
+ * @returns ????????????
+ */
 function update_filename_at_DB(db, padId, newName) {
```
Comment (ManosKokos) [inline src/Controller/Controller.js]: This either returns null or end the function, without return statement so returns nothing by default.
In line:184 `if (result === null) ` result will be undefined and it won't return 500 but is this ok??

Comment (parriiss) [reply src/Controller/Controller.js]: So when somethings wrong in update then it returns null but if everything ok then it returns undefined.I will change it to return something else more relevant

**Pull request merged**